tests/testthat/_snaps/conditions/abort-error.txt

```diff
@@ -0,0 +1,24 @@
+Error in `h()`:
+! !
+Backtrace:
+     ▆
+  1. ├─evaluate::evaluate(...)
```

Comment:
It feels like we're at a good place to merge this PR, but I do wonder if something (rlang?) should do more to automatically trim this stack.

Reply:
I included this for the purpose of discussing exactly this. You commented before I made any comments :) 

My understanding is that it would not be rlang, but it would be evaluate to indicate to rlang which is the top env using `rlang_trace_top_env` option value. 

This is one of the thing Lionel told me to do on knitr to have clean traceback by default in **knitr**. But maybe this should be done at evaluate level, to always have a trimmed trace. 

I added a test as example for this in one of the R file.

Reply:
I think this would be done with c9d7a69